editor/meta-boxes/meta-boxes-iframe/index.js

```diff
@@ -12,9 +13,12 @@ import { addQueryArgs } from '@wordpress/url';
 import { Component } from '@wordpress/element';
 
 /**
- * Internal dependencies.
+ * Internal dependencies
  */
-import MetaBoxPanel from './panel.js';
+import './style.scss';
+import './meta-box-iframe.scss';
```

Comment:
I see this is moved from what existed previously but... should we be importing this here? There are styles applied to a `body` selector, and it seems like they're meant only in the context of the iframe partial frame body:

https://github.com/WordPress/gutenberg/blob/653e5556d1b16f87ae1c1c16eea45afb55e78fff/lib/meta-box-partial-page.php#L158-L163

Reply:
I think it still needs to be imported somewhere, so that webpack can recognize it. Not sure where else it would be relevant. Is it possible to get webpack to build files that are not imported?

Reply:
Still thinks it should stay here since the iframe is rendered by this component.